lib/locales.js

```diff
@@ -16,16 +16,16 @@ exports.normalizeLocale = normalizeLocale;
 
 // These are the exceptions to the default algorithm for determining a locale's
 // parent locale.
-var PARENT_LOCALES_HASH = require('../data/supplemental/parentLocales.json')
+var PARENT_LOCALES_HASH = require('cldr-core/supplemental/parentLocales.json')
     .supplemental.parentLocales.parentLocale;
 
-var PLURAL_LOCALES_HASH = require('../data/supplemental/plurals.json')
+var PLURAL_LOCALES_HASH = require('cldr-core/supplemental/plurals.json')
     .supplemental['plurals-type-cardinal'];
 
 var DATE_FIELDS_LOCALES_HASH = glob.sync('*/dateFields.json', {
-    cwd: path.resolve(__dirname, '..', 'data', 'main'),
+    cwd: p.resolve(__dirname, '..', 'node_modules', 'cldr-dates-full', 'main'),
```

Comment:
this might fail in old versions of node/npm if another module attempt to install cldr-dates-full, my recommendation is to do something like `p.resolve(require.resolve('cldr-dates-full/package.json'), './main');`